### PR TITLE
Wappalyzer Minor Code Cleanup

### PR DESCRIPTION
**Pull request by kingthorin**
- Remove unnecessary code comments.
- Remove some extra lines.
- Remove some default/boilerplate JavaDoc content.
- Remove one un-necessary Warning Suppression.

- WappalyzerPassiveScanner address TODO in overridden method
appliesToHistoryType accept core handling for getDefautlHistoryTypes.
- ZapAddOn.xml update changes section.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/wappalyzer/Application.java [hunk 1]
```diff
@@ -117,9 +117,5 @@ public void setIcon(ImageIcon icon) {
 	}
 
 	// TODO global js variables - cant use?
-	//private Pattern env;
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/wappalyzer/Application.java]: The above comment should be removed as well?

File: src/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java [hunk 2]
```diff
@@ -165,20 +163,9 @@ public void setLevel(AlertThreshold level) {
...
-		switch (historyType) {
-		case HistoryReference.TYPE_PROXIED:
-		case HistoryReference.TYPE_ZAP_USER:
-		case HistoryReference.TYPE_SPIDER:
-		case HistoryReference.TYPE_SPIDER_AJAX:
-			return true;
-		default:
-			return false;
-		}
+		 return PluginPassiveScanner.getDefaultHistoryTypes().contains(historyType);
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java]: - The minimum ZAP version ("not-before-version" in ZapAddOn.xml file) should be updated accordingly;
 - There's an extra white space before "return" that could be removed.

File: src/org/zaproxy/zap/extension/wappalyzer/ZapAddOn.xml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Technology detection using Wappalyzer</name>
-	<version>8</version>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/wappalyzer/ZapAddOn.xml]: Version 8 was not yet released(?).

**Comment by thc202** [comment]
Code changes look good to me, although there's some unrelated changes in `pscanrulesAlpha/ZapAddOn.xml`.
In the commit message "increment version" should be removed.

**Comment by kingthorin** [comment]
Fixed.

**Comment by thc202** [comment]
Looks good to me, thank you!

**Pull request merged**